libaleth-interpreter/VM.cpp

```diff
@@ -163,10 +130,10 @@ void VM::adjustStack(int _required, int _change)
 
     // adjust stack and check bounds
     if (m_stackEnd < m_SP + _required)
-        throwBadStack(_required, _change);
+        throwBadStack(_required);
```

Comment:
Can we make two separate functions for underflow and overflow? Then we won't need an `if` inside throw function and it can be static as others